src/messages/Video/Video.tsx

```diff
@@ -62,6 +62,17 @@ const Video: FC = () => {
 		}
 	}, [playing]);
 
+	const videoCaptionsConfig = {
+		tracks: [
+			{
+				kind: "subtitles",
+				src: captionsUrl as string,
+				srcLang: "en-US", // TODO: Get language from Say node in the future
+				label: "English",
```

Comment:
Should we change the label to "CC" or "Closed Captions"?

Reply:
Menu item was already named 'Captions' with a CC icon. the label was displayed below that when the caption was turned on.